wp-cldr.php

```diff
@@ -10,6 +10,138 @@
  * Domain Path: /languages/
  * License:     GPLv2 or later
  * License URI: https://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * @package    wp-cldr
  */
 
 require_once plugin_dir_path( __FILE__ ) . 'class.wp-cldr.php';
+require_once( ABSPATH . 'wp-admin/includes/translation-install.php' );
+
+add_action( 'admin_menu', 'wp_cldr_menu' );
+
+/**
+ * Gets the wp-admin menu info.
+ */
+function wp_cldr_menu() {
+	add_options_page(
+		'WP CLDR Options',
+		'WP CLDR',
+		'manage_options',
+		'wp-cldr',
+		'wp_cldr_settings'
+	);
+}
+
+/**
+ * Gets the settings and examples page for the plugin.
+ */
+function wp_cldr_settings() {
+
+	if ( isset( $_GET['locale'] ) && 1 < strlen( sanitize_text_field( wp_unslash( $_GET['locale'] ) ) ) ) {
+		$locale = sanitize_text_field( wp_unslash( $_GET['locale'] ) );
```

Comment:
instead of sanitizing / unslashing these, could we just do whitelist checks? If it isn't in the list of things we know about, use the default.